subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/CachedKotlinTaskExecutionIntegrationTest.groovy

```diff
@@ -53,6 +53,10 @@ class CachedKotlinTaskExecutionIntegrationTest extends AbstractIntegrationSpec i
             }
         """
         when:
+        // Distribution needed to trigger MixInCoreTypesTransformingClassLoader
+        // injecting necessary methods with internal return types, because
+        // Kotlin compiler plugin was compiled against Gradle 3.2
+        executer.requireGradleDistribution()
```

Comment:
Moved.

@bamboo is aware of this needed to be removed once we have a new version of the Kotlin compiler plugin.